packages/sdks/src/components/blocks/blocks.types.ts

```diff
@@ -6,9 +6,13 @@ import type {
 import type { BlocksWrapperProps } from './blocks-wrapper.lite';
 
 export type BlocksProps = Partial<
-  Omit<BlocksWrapperProps, 'BlocksWrapper' | 'BlocksWrapperProps'>
+  Omit<
+    BlocksWrapperProps,
+    'BlocksWrapper' | 'BlocksWrapperProps' | 'classNameProp'
+  >
 > & {
   context?: Signal<BuilderContextInterface>;
   registeredComponents?: RegisteredComponents;
   linkComponent?: any;
+  className?: string;
```

Comment:
if you rename `classNameProp` to `className`, you should be able to undo the changes in this file, since the `className` prop in both cases will match, so it will flow through.